src/client/Microsoft.Identity.Client/ApiConfig/Executors/ConfidentialClientExecutor.cs

```diff
@@ -133,22 +202,22 @@ public async Task<Uri> ExecuteAsync(
 
         public async Task<AuthenticationResult> ExecuteAsync(
             AcquireTokenCommonParameters commonParameters,
-            AcquireTokenByUsernamePasswordParameters usernamePasswordParameters,
+            AcquireTokenByUsernamePasswordParameters userNamePasswordParameters,
```

Comment:
Is this some IntelliSense feature that changed this variable? We generally use username as a single word.